docs/add-node.md

```diff
@@ -0,0 +1,38 @@
+# Add Node to Existing Pool
+
+## Node preparation
+
+1. Add this files from running node:
+```
+/var/lib/indy/network_name/pool_transactions_genesis
+/var/lib/indy/network_name/domain_transactions_genesis
+```
+
+2. Initialize keys, aliases and ports on new nodes using init_indy_node script.
+Example:
+```
+sudo su - indy -c "init_indy_node NewNode 0.0.0.0 9701 0.0.0.0 9702 0000000000000000000000000NewNode"
+```
+
+3. Starting the node the first time will read in the pool_transactions_sandbox and transactions_sandbox files into the ledger. It only reads it the one time so make sure the first are correct before starting the service.
+```
+sudo systemctl start indy-node
+sudo systemctl status indy-node
+sudo systemctl enable indy-node
+```
+
+## Add Node to the Pool
+
+1. As Trustee add another Steward if needed (only Steward can add new Validator, and only one)
```

Comment:
(only Steward can add a new Validator Node; a Steward can add one and only one Validator Node)